diff --git a/hw.txt b/hw.txt
new file mode 100644
index 0000000..95d09f2
--- /dev/null
+++ b/hw.txt
@@ -0,0 +1 @@
+hello world
\ No newline at end of file
diff --git a/src/category/controllers/category.controller.ts b/src/category/controllers/category.controller.ts
index 87f79ef..bfaba7c 100644
--- a/src/category/controllers/category.controller.ts
+++ b/src/category/controllers/category.controller.ts
@@ -21,7 +21,7 @@ export class CategoryController {
         }
     }
 
-    async getCategoryById(req: Request, res: Response) {
+    async getCategoryById(req: Request, res: Response): Promise<Response> {
         const { id } = req.params
         try {
             const data = await this.categoryService.findByCategoryId(id)
